src/ThreeEditor/js/Strings.js

```diff
@@ -508,7 +508,7 @@ function Strings(config) {
 			'sidebar/geometry/plane_geometry/heightsegments': 'Segments en Hauteur',
 
 			'sidebar/geometry/ring_geometry/innerRadius': 'Rayon intérieur',
-			'sidebar/geometry/ring_geometry/outerRadius': 'Rayon extérieur',
+			'sidebar/geometry/ring_geometry/radius': 'Rayon extérieur',
```

Comment:
We are not planning to support languages other than English, can you remove other languages completely?